root/programs/C#/Frameworks/Infrastructure/Public/IO/ASymmetricCryptography.cs

```diff
@@ -100,16 +102,20 @@ public static byte[] EncryptBytes(byte[] source, string publicKey)
             // RSACryptoServiceProviderオブジェクトの作成
             RSACryptoServiceProvider rsa
                 = (RSACryptoServiceProvider)RSACryptoServiceProvider.Create(); // devps(1703)
-            
+
             // 公開鍵
             rsa.FromXmlString(publicKey);
 
-            // 暗号化する
+            // 暗号化する（XP以降の場合のみ2項目にTrueを指定し、OAEPパディングを使用できる）
             byte[] temp = rsa.Encrypt(source, false);
             rsa.Clear(); // devps(1725)
```

Comment:
@daisukenishino 

Isn't it necessary to delete the key entry in the container?
See [MSDN](https://msdn.microsoft.com/en-us/library/tswxhw92(v=vs.110).aspx).